magprime/templates/emails/dealer_deadline_change.txt

```diff
@@ -1,11 +0,0 @@
-{{ group.leader.first_name }},
```

Comment:
While I was making changes and reorganizing the automated emails anyway, I deleted all of the emails from last year which are no longer relevant.  This is basically just a couple of one-off emails explaining deadline extensions or correcting mistaken emails which went out with typos and such.